cpp/tests/CMakeLists.txt

```diff
@@ -408,6 +414,19 @@ ConfigureTest(LEGACY_CSV_TEST "${LEGACY_CSV_TEST_SRC}")
 ConfigureTest(LEGACY_JSON_TEST "${LEGACY_JSON_TEST_SRC}")
 ConfigureTest(LEGACY_ORC_TEST "${LEGACY_ORC_TEST_SRC}")
 
+# Kafka integration tests are disabled by default.
+if(ENABLE_KAFKA_TESTS)
+    #include_directories("${RDKAFKA_INCLUDE}")
```

Comment:
Delete commented out code.

Reply:
good catch. I was testing some CI things and forgot to uncomment that code. Fixed.